apps/user_status/lib/Connector/UserStatus.php

```diff
@@ -56,8 +57,8 @@ public function __construct(Db\UserStatus $status) {
 		$this->message = $status->getCustomMessage();
 		$this->icon = $status->getCustomIcon();
 
-		if ($status->getStatus() === 'invisible') {
-			$this->status = 'offline';
+		if ($status->getStatus() === StatusService::INVISIBLE) {
+			$this->status = StatusService::OFFLINE;
```

Comment:
```suggestion
		if ($status->getStatus() === IUserStatus::INVISIBLE) {
			$this->status = IUserStatus::OFFLINE;
```

Invisible is missing from the public interface, but I guess it also makes sense